tso-server/src/test/java/org/apache/omid/tso/TestPersistenceProcessor.java

```diff
@@ -145,7 +145,7 @@ public void testCommitPersistenceWithSingleCommitTableWriter() throws Exception
 
         ObjectPool<Batch> batchPool = spy(new BatchPoolModule(tsoConfig).getBatchPool());
 
-        ReplyProcessor replyProcessor = new ReplyProcessorImpl(metrics, panicker, batchPool);
+        ReplyProcessor replyProcessor = new ReplyProcessorImpl(tsoConfig, metrics, panicker, batchPool);
```

Comment:
According to the changes above regarding to WaitStrategy injections, the *ProcessorImpl clases in all the tests should create the objects of the previous WaitStrategy in each case. This is the matching:

PersistenceProcessorImpl - BlockingWaitStrategy
ReplyProcessorImpl - BlockingWaitStrategy
RetryProcessorImpl - YieldingWaitStrategy

e.g. here should be:

```java
ReplyProcessor replyProcessor = new ReplyProcessorImpl(new BlockingWaitStrategy(), metrics, panicker, batchPool);
```

The same applies to the other *ProcessorImpl instantiations below.